.github/workflows/wash.yml

```diff
@@ -26,7 +20,29 @@ permissions:
   contents: read
 
 jobs:
+  should_run:
+    runs-on: ubuntu-latest
+    outputs:
+      modified: ${{ steps.changes.outputs.wash }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            wash:
+              - './.github/workflows/wash.yml'
+              - 'Cargo.lock'
+              - 'Cargo.toml'
+              - 'crates/control-interface/**'
+              - 'crates/core/**'
+              - 'crates/secrets-types/**'
+              - 'crates/wash-cli/**'
+              - 'crates/wash-lib/**'
```

Comment:
If we really wanted to get specific here, I could change these to be only actual things that affect functionality.

E.g., instead of
```yaml
- 'crates/wash-cli/**'
```

It could be:

```yaml
- 'crates/wash-cli/Cargo.toml'
- 'crates/wash-cli/build.rs'
- 'crates/wash-cli/src/**'
- 'crates/wash-cli/tests/**'
```

This would basically optimize the CI to not re-run the workflow if we changed the wash README / changelog / Makefile / devcontainer.

Reply:
I would exclude readme, changelog, and devcontainer